src/controllers/ProductsController.js

```diff
@@ -415,7 +415,6 @@ class ProductsController extends Controller {
 
     const devices = (await this._deviceAttributeRepository.getManyFromIDs(
       deviceIDs,
-      this.user.id,
```

Comment:
user.id in our case is always admin here, but product devices always belong to somebody else, in order to find them we don't want to provide id.